release-notes/2.1/Preview/2.1.0-preview2-known-issues.md

```diff
@@ -26,4 +26,114 @@ We have temporarily removed the Preview 2 installers from the Linux package feed
+
+2. Instead of adding `<PackAsTool>true</PackAsTool>` property in project file, pass it to `pack` via command line. Remove the property from the .csproj file and use `dotnet pack /p:PackAsTool=true`.
+
+3. Create a separate project just to pack. Move all of the logic you intend to test into a project that this pack project references. Include the `PackAsTool=true` property only in the pack project, allowing you to test other projects.
+
+### Breaking changes from .NET Core SDK 2.1.300 Preview 1 Global Tools
+
+.NET Core SDK 2.1.300 Preview 1 provided feedback that led to changes in Global Tools. There are several breaking changes between Preview 1 and Preview 2, and the recommendation is:
+
+**Action**
+* Tool authors should rebuild and republish Global Tools using SDK 2.1.300 Preview 2 or higher, targeting .NET Core Runtime 2.1.0 with Preview 2 of the runtime also installed on the machine (the normal case).
+* Tool users should 
+  * Upgrade to .NET Core 2.1.0 Preview 2 or higher.
+  * Remove the tools packages folder.
+  * Remove the tools folder, empty it, or remove individual Preview 1 tools.
+  * Reinstall upgraded tools.
+
+Tool users can remove all tools
+
+**Windows**
+
+1. Delete the tool packages directory: `%USERPROFILE%\.dotnet\toolspkgs`.
+2. Delete the tools, empty it or remove individual Preview 1 tools from the directory: `%USERPROFILE%\.dotnet\tools` (`C:\Users\NAME\.dotnet\tools`).
+
+**MacOS and Linux**
+
+1. Delete the tools directory: `$HOME/.dotnet/tools`.
+2. Delete the tools, empty it or remove individual Preview 1 tools from the directory: `$HOME/.dotnet/toolspkgs`.
+
+If you would like to continue to use Preview 1 Global Tools, you may encounter these problems:
+
+#### Preview 1 Global Tools give warning when install in later SDKs
+
+If a Global Tool created in Preview 1 is installed in the Preview 2 or later SDK, you will receive the following warning: 
+
+`Format version is missing. This tool may not be supported in this SDK version.`
+
+To help with future forward compatibility, Global Tools now include a format version for the internal structure they were created with. If this format version is not present or changes, this warning will be displayed. The format version is not present in Preview 1 Global Tools.
+
+**Action -** You may ignore this warning moving from Preview 1 to later versions of 2.1.300.
+
+#### Preview 1 Global Tools cannot be uninstalled with later previews
+
+The `dotnet tool uninstall` command in Preview 2 cannot uninstall Global Tools that were installed using Preview 1 due to a change in the installation layout. The location of package content (the implementation) moved from `$HOME/.dotnet/toolspkgs` to `$HOME/tools/.store`. The Preview2 SDK will try to delete package content from `$HOME/tools/.store` where it did not exist. 
```

Comment:
$HOME/tools/.store -> $HOME/.dotnet/tools/.store

Sorry I think I sent you this. And I missed /.dotnet in some case

Reply:
There is 2 places 

moved from `$HOME/.dotnet/toolspkgs` to `$HOME/!!HERE!!/tools/.store`. The Preview2 SDK will try to delete package content from `$HOME/!!HERE!!/tools/.store` where it did not exist.